JCenter not reachable, can't build on Android

Hi,
I'm using the latest (`1.1.0`) version of this package, and when trying to build and run the app on Android, I get the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':react-native-exit-app'.
> Could not resolve all artifacts for configuration ':react-native-exit-app:classpath'.
   > Could not resolve com.android.databinding:baseLibrary:3.1.4.
     Required by:
         project :react-native-exit-app > com.android.tools.build:gradle:3.1.4 > com.android.tools.build:gradle-core:3.1.4 > com.android.databinding:compilerCommon:3.1.4
      > Could not resolve com.android.databinding:baseLibrary:3.1.4.
         > Could not get resource 'https://jcenter.bintray.com/com/android/databinding/baseLibrary/3.1.4/baseLibrary-3.1.4.jar'.
            > Could not HEAD 'https://jcenter.bintray.com/com/android/databinding/baseLibrary/3.1.4/baseLibrary-3.1.4.jar'.
               > Read timed out
```

.. which makes sense because https://jcenter.bintray.com/com/android/databinding/baseLibrary/3.1.4/baseLibrary-3.1.4.jar is not reachable any more (403 Forbidden).

What can I do about this? I'm thankful for any hints. Thanks!

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -5,7 +5,7 @@ buildscript {
     if (project == rootProject) {
         repositories {
             google()
-            jcenter()
+            mavenCentral()
         }
 
         dependencies {

